CHANGES/9137.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+Added :exc:`ClientConnectionResetError`. Client code that previously threw :exc:`ConnectionResetError`
+will now throw this -- by :user:`Dreamsorcerer`.
```

Comment:
Should we tell them it inherits from `ConnectionResetError` so they know its backwards compat?

Reply:
I figured they'd look at the exception in the docs, where this is shown. The main information I'm trying to convey here is that if they were catching ConnectionResetError as per the linked issue, then they can now remove that as it'll be caught as a regular ClientException.